std_msgs/Header header
int16 isBlocking
geometry_msgs/Vector3 avoid
float32 dt
